content/blog/technical-assessment-evaluation-criteria.md

```diff
@@ -0,0 +1,104 @@
+
+## Software Craft Round
+Congratulations! You have gotten to this round. Here is where all the fun happens. You will fire up your IDE with the language of your choice and a simple Hello World program pre-installed and get ready to get your hands dirty. We will solve a short Kata together as a team! Consider this round as a pairing exercise, in which an Incubyte developer will partner with you and act as a navigator. It is fine to look stuff up on the internet. We understand that having Maven or Gradle dependency coordinate on top of the code is difficult. (We google things while working on production projects ALL the time!)
+
+This brings us to the end of our interview process, expect to walk away with an offer once you have cleared all these rounds. 
+
+Now comes the important part, how to clear this interview round? We have broken down our requirements into 2 parts:
+1. Technical
+2. Non-technical  
+
+## Technical
+We'd love to see how you code and how you think. What matters to you and how does it align with what matters to us. Here are some of the technical things that matter most to us:
+  
+### Build tools like Maven or Gradle
+
+We believe that build tools are first-class members of our development life cycle. Choosing the best build and dependency management tool when beginning a new project is vital and can go a long way.
+
+### Clean coding practices
+
+It is not possible to evaluate everything during an interview, but below mentioned are a few key indicators that give us an insight into how you work:
+
+#### Code format
+
+Code writing skills are essential, but so is code readability. A neatly formatted code is readable.
+
+#### Amount of nesting in your code
+
+Excessive nesting makes the code difficult to understand and raises cognitive complexity.
+
+#### Applying best practices to your code:
+
+Using clean code practices speaks much more of a personal habit as much as your coding skill. It can range from using the appropriate naming conventions to Page Object Models if you are working on test automation. Having some linters integrated into your IDE can help you adhere to best practices.
+
+#### Commented-out code
+
+Commented code is as good as a dead code. Version control systems, as well as modern IDEs, are great at maintaining history, code should not be kept commented with your current production code.
+
+### Other attitudes we recommend
+
+#### XP Practices
+
+TDD, Pairing, Refactoring, and working in iterations are a few practices we would love people to try. We take TDD very seriously.
+
+We do not expect everyone to be proficient at it, but learning and knowing about it will be a great plus. Preparing for the interview by doing a few Katas will help a lot. BDD is a preferred method if you are programming for Automation Testing.
+
+## The key pointers when we hunt for the best minds are
```

Comment:
This entire section seems redundant and also not comprehensive. Should we remove it?